### ISSUE TITLE
color picker - not picking the color I'm clicking on when site is scrolled

### ISSUE THREAD

**Issue by Arcanst** [primefaces/primeng#3977]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
No need for plunkr, just go to the documentation of color picker https://www.primefaces.org/primeng/#/colorpicker .

**Current behavior**
When the site is fully scrolled up like this:
![obraz](https://user-images.githubusercontent.com/17785684/30539773-4eee157c-9c74-11e7-9d94-508ab8697c69.png)
and you pick the color from whichever color picker, everything works just fine. When you scroll the site just a little bit down and you try picking a color, the circle of picked color doesn't appear where I clicked. There is some kind of offset that grows when you scroll the site.

**Expected behavior**
There should be no offset, the color clicked should be the color picked.

**Minimal reproduction of the problem with instructions**
Just go to the documentation site: https://www.primefaces.org/primeng/#/colorpicker .

Best regards,
Daniel

**Comment by ASK83** [comment]
I am experiencing a similar issue. And gets to its worst when you have a code that hides the top navigation based on the scroll. Basically the color picker would be off by the amount of scroll in the Y direction, which makes it unusable.

**Comment by scolt** [comment]
I've had the same issue.

**Comment by jharker** [comment]
We've had the same issue -- it really makes the color picker unusable on any kind of page where things move, which is pretty much all the time.

Does anyone know a workaround for this?

**Comment by jharker** [comment]
Hey, all -- I just saw that it appears a fix for this was merged three days ago:

https://github.com/primefaces/primeng/commit/a24b44ea2014e099ce9a0da6724e6507367796f9

Hope this is helpful.

**Comment by cagataycivici** [comment]
Fixed in 4.2.2.
